src/components/editTestCase/qdm/RightPanel/RightPanel.tsx

```diff
@@ -53,6 +55,23 @@ const RightPanel = ({
         {activeTab === "details" && (
           <DetailsSection canEdit={canEdit} measureName={measureName} />
         )}
+
+        {activeTab === "measurecql" &&
+          (!cqlErrors ? (
+            <div data-testid="test-case-cql-editor" id="test-case-cql-editor">
+              <MadieEditor
+                value={measureCql}
+                height="52vh"
+                readOnly={true}
+                validationsEnabled={false}
+              />
```

Comment:
Can we also address the 3rd AC from the Story ?
ensuring that the view only area is horizontally scrollable

Reply:
Removed after discussion